block_element.go

```diff
@@ -279,6 +279,7 @@ type MultiSelectBlockElement struct {
 	InitialChannels      []string                  `json:"initial_channels,omitempty"`
 	MinQueryLength       *int                      `json:"min_query_length,omitempty"`
 	MaxSelectedItems     *int                      `json:"max_selected_items,omitempty"`
+	Filter               *SelectBlockElementFilter `json:"filter,omitempty"`
```

Comment:
[nits]
We should rename to `ConversationListsFilter`.

Reply:
Hi @kanata2 i've updated to be `ConversationListsFilter`, kindly help to check again :)

Reply:
(Sorry, this suggestion is backward imcompatible change... I'll make new minor version and include this.)

Reply:
Btw is there anything that i need to do ? or just waiting for this MR merged by you? @kanata2

Reply:
I'll merge and release as soon as it's ready.

Reply:
Sorry, this was a mis-communication; it should be `Filter` -> `ConversationListFilter`. We definitely don't want to rename the existing `SelectBlockElementFilter`. If you want to correct this and the linting issues I'll have another look.